content/docs/command-reference/repro.md

```diff
@@ -157,8 +157,9 @@ up-to-date and only execute the final stage.
   corresponding pipelines, including the target stages themselves. This option
   has no effect if `targets` are not provided.
 
-- `--pull` - try automatically [pulling](/doc/command-reference/pull) missing
-  cache for outputs restored from run-cache.
+- `--pull` - [pulls](/doc/command-reference/pull) dependencies and outputs
+  involved in the stages being reproduced, if they are found in remote storage.
+  Note that it checks the run-cache too (available history of stage runs).
```

Comment:
A few questions about this still remain (see https://github.com/iterative/dvc.org/pull/1841#discussion_r508696705).

Reply:
Answers in https://github.com/iterative/dvc.org/pull/1841#discussion_r508804055! Resolving